### PR TITLE
README.md updated to fit 2.0.1

### PR DESCRIPTION
**Pull request by WoZ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+};
+
+const monitor = new ConsulKvMonitor(monitorConfig, consulClient);
+
+function printData(consulKvData) {
+    const keys = consulKvData.getKeys();
+    keys.forEach(key => {
+        let value = consulKvData.getValue(key);
+        if (value === Object(value)) {
+            // we received json decoded data, so just stringifying to pretty outpur in this example
```
Comment (oleh-poberezhets) [inline README.md]: Bug in word 'outpur'

File: README.md [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+```
+
+And finally delete all keys:
+
+```shell
+$ consul kv delete lcm/server-2
+Success! Deleted key: lcm/server-2
+```
+
+Monitor emits `change` event with an object without any keys.
```
Comment (oleh-poberezhets) [inline README.md]: Monitor emits `changed`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,331 @@
+# consul-kv-monitor
+
+[![NPM version](https://img.shields.io/npm/v/consul-kv-monitor.svg)](https://www.npmjs.com/package/consul-kv-monitor)
+[![Build Status](https://travis-ci.org/LCMApps/consul-kv-monitor.svg?branch=master)](https://travis-ci.org/LCMApps/consul-kv-monitor)
+[![Coverage Status](https://coveralls.io/repos/github/LCMApps/consul-kv-monitor/badge.svg?branch=master)](https://coveralls.io/github/LCMApps/consul-kv-monitor?branch=master)
+
+Module monitors consul key-value store and notifies about changes of a value or a set of values. It relies on
```
Comment (Chertowski) [inline README.md:7]: This module

**Review by oleh-poberezhets — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (WoZ) [reply README.md:7]: Done

**Pull request merged**